release/models/gribi/openconfig-gribi.yang

```diff
@@ -0,0 +1,39 @@
+module openconfig-gribi {
+  yang-version "1";
+
+  prefix "oc-gribi";
+
+  namespace "http://openconfig.net/yang/gribi";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system-grpc { prefix oc-grpc; }
+
+  organization
+    "OpenConfig Working Group";
+
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module defines a set of exdtensions that provide gRIBI (the gRPC
+    RIB Interface) specific extensions to the OpenConfig data models.
```

Comment:
nit: double use of extensions in a single statement :) Maybe gRIBI specific model?